docs/github/v2/15-deployment/ios.md

```diff
@@ -141,7 +175,13 @@ Then change the build_app section so that it uses the new `xcworkspace`:
     )
 ```
 
-### 4- Add jobs to your GitHub workflow
+### 4- Add jobs to your GitHub Actions workflow
+
+Building for iOS requires two steps: Unity builds your project and generates an Xcode project, which then must be built in Xcode via Fastlane. This workflow below splits that across two jobs: the Unity build happens on Linux (which is cheaper per minute), uploads the generated Xcode project as an artifact to the Action run, and then downloads that Xcode project on a Mac runner to finish up the build from there. If your project requires IL2CPP, you will need to complete the Unity build on a Mac runner.
```

Comment:
I have not yet tested unity-builder Mac runner support. Is this mature enough to recommend, and if so, are there additional steps required beyond just changing the runner type?

Reply:
I've been using the Mac Runner for my IL2CPP Mac builds, and it's been stable for me. I know a couple of other users have been using it as well, and I haven't seen any issues reported. I think recommending it is good, and yes, no additional change is required by the user beyond changing the `runs-on`.